### DISCUSSION TITLE
Is there a way to use an env in place of the token?

### DISCUSSION THREAD

**Discussion by ooker777** [ObsidianPublisher/obsidian-github-publisher#157] [General]
It would be safer and more convenient to be able to do so

**Comment by Mara-Li** [comment]
I can maybe store the key on a file in the `.obsidian` folder, and crypto it using the Obsidian module, but I'm afraid I can't do more...

**Reply by Mara-Li** [reply]
Discuss this in #160 ! :)

**Reply by ooker777** [reply]
thanks for your work. It must be a burden to you to support this plugin

**Reply by Mara-Li** [reply]
The only problem is when I take some times away of my pc and I get like 3 bug report and nothing in month lol!

But it's pretty fun to be honest so it's okay.
